tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1095,22 +1095,174 @@ public function test_build_line_items( bool $itemized = false, array $expected_i
+
+		if ( isset( $product ) && $product ) {
+			$product->delete( true );
+		}
+
+		if ( $has_block ) {
+			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited -- Restoring $post after test isolation.
+			$post = $saved_post;
+		}
+
+		$this->assertSame( $expected, $actual );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_adaptive_pricing_supported is not named in camelCase. (undefined)

(CamelCaseMethodName)